meetings/2022-02-28.md

```diff
@@ -0,0 +1,131 @@
+* EP: OAuth and OIDC do rely on TLS, so we'd be aligned with this, but we could be putting extra burden on future approaches eg DID. The requirement would apply to the publishing-side rather than consuming parties (e.g. on webid provider, but not on OP)
+* AC: What we currently have is "... must be transported over TLS", which is in the domain of the OP. When it comes to "external" resources, it leaks out of the scope of Solid-OIDC. The Solid-OIDC spec should not add additional constaints on the WebID spec, even when they make sense.
+* KD: How should the recommendations be linked so that developers don't miss them ?
+* AC: There is always a tension between restating things from other specs or duplicating text. As long as you refer normatively I don't think we need to restate details from other specifications. Security considerations are not normative but we use normative language like MUST. I see us being inconsistent. 
+* EP: What about the Client Identifier, it being defined by Solid-OIDC
+* AC: In that case it would make a lot of sense for it to be required to use https
+* EP: The WebID spec doesn't specify, so in this case is it bad to add our own requirements on top, narrowing it of some sorts, like requiring the trusted OP to be referred?
+
+### Primer: multiple issues
+
+[Issue#83](https://github.com/solid/solid-oidc/issues/83)
+
+* EP: Contributions for [Dimitri Teleguin](https://github.com/dteleguin) (i believe keycloak contributor). If anyone can help him contribute that would be great.
+* AC: Indeed, the primer has been neglected. I'll open some small PRs to try and address some issues.
+* EP: Very well, i'll focus on #65. KD, do you have feedback on the primer and spec ?
+* KD: Not necessarily, I'm learning a lot still.
+
+### Organize sections by conformance classes
+
+[Issue#65](https://github.com/solid/solid-oidc/issues/65)
+
+* EP: First step with PR#79 for the WebID profile document. Similar example is the ID token verification, which happens on both the client side and the AS too. How do we want to eventually add the test suite annotations on the conformance classes ? In the case of multiple parties who need to comply with a certain part, eg ID token verification. Could it be identified as an atomic requirement, applied to multiple conformance classes ?
+* AC: This kind of spec organization will make things easier for implementers and people testing implementations. Because there are a number of conceptual entites involved, each having its own responsibilities. Unless you understand each of these components, it is confusing who is responsible for what. The new organization intends at reducing this confusion. KD, since you're looking at implementing an OP your feedback will be very valuable.
```

Comment:
```suggestion
* AC: This kind of spec organization will make things easier for implementers and people testing implementations. Because there are a number of conceptual entites involved, each having its own responsibilities. Unless you understand each of these components, it is confusing who is responsible for what. The new organization intends to reduce this confusion. KD, since you're looking at implementing an OP, your feedback will be very valuable.
```